sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/cosmos/springdata/SpringDataCosmosIT.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.integration.tests.cosmos.springdata;
+
+import com.azure.spring.cloud.integration.tests.cosmos.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@ActiveProfiles("cosmos-springdata")
+public class SpringDataCosmosIT {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SpringDataCosmosIT.class);
+    private final String userId = "testSpringDataCosmos";
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Test
+    public void testSpringDataCosmosOperation() {
+        LOGGER.info("SpringDataCosmosIT begin.");
+        User testUser = new User(userId, "testFirstName", "testLastName", "test address line one");
+        userRepository.save(testUser).block();
```

Comment:
If you are using `block` here, you can use non-reactive repository instead of the `ReactiveCosmosRepository`. So we can remove the `block()` call here.